src/plugins/execution-time-estimator-navgraph/navgraph_estimator.h

```diff
@@ -45,6 +44,6 @@ class NavGraphEstimator : public ExecutionTimeEstimator
 	float                    last_pose_x_;
 	float                    last_pose_y_;
 	float                    speed_;
+	constexpr static char    cfg_prefix_[] = "plugins/execution-time-estimator/navgraph";
```

Comment:
A forward slash is missing here:
```suggestion
	constexpr static char    cfg_prefix_[] = "plugins/execution-time-estimator/navgraph/";
```